docs/examples/buildConfigs/ruby-hello-world.yml

```diff
@@ -0,0 +1,44 @@
+controller: DeploymentConfig
+name: ruby
+containers:
+- image: ""
+triggers:
+- imageChangeParams:
+    automatic: true
+    containerNames:
+    - ruby
+    from:
+      kind: ImageStreamTag
+      name: origin-ruby-sample:latest
+  type: ImageChange
+services:
+- portMappings:
+  - "8080"
+imageStreams:
+- name: origin-ruby-sample
+- name: ruby-20-centos7
+  tags:
+  - name: latest
+    from:
+      kind: DockerImage
+      name: openshift/ruby-20-centos7
+buildConfigs:
+- triggers:
+  - type: "ImageChange"
+  source:
+    type: "Git"
+    git:
+      uri: "https://github.com/openshift/ruby-hello-world"
+    dockerfile: "FROM openshift/ruby-22-centos7\nUSER example"
```

Comment:
again this doesn't seem right :-(

Reply:
@kadel posted above, this is allowed with git.

Reply:
I think we can remove this from this example to make it more concise maybe